src/Entity/User.php

```diff
@@ -220,6 +220,30 @@ public function setDeletedAt(?\DateTime $deletedAt): User
         return $this;
     }
 
+    public function addProgramme(Programme $programme): self
+    {
+        if ($this->programmes->contains($programme)) {
+            return $this;
+        }
+
+        $this->programmes->add($programme);
+        $programme->addCustomer($this);
+
+        return $this;
+    }
+
+    public function removeProgramme(Programme $programme): self
+    {
+        if ($this->programmes->contains($programme)) {
+            return $this;
+        }
```

Comment:
this is bad logic, if you want to remove a programme, for sure you would the programme collection to have it. I think you are missing a `!` in front of your condition